okcurl/src/main/resources/META-INF/native-image/okhttp3/okcurl/reflect-config.json

```diff
@@ -0,0 +1,45 @@
+[
+  {
+    "name" : "okhttp3.curl.Main",
+    "allDeclaredConstructors" : true,
+    "allPublicConstructors" : true,
+    "allDeclaredMethods" : true,
+    "allPublicMethods" : true,
+    "fields" : [
+      { "name" : "allowInsecure" },
+      { "name" : "callTimeout" },
+      { "name" : "completionScript" },
+      { "name" : "connectTimeout" },
+      { "name" : "data" },
+      { "name" : "followRedirects" },
+      { "name" : "headers" },
+      { "name" : "method" },
+      { "name" : "readTimeout" },
+      { "name" : "referer" },
+      { "name" : "showHeaders" },
+      { "name" : "showHttp2Frames" },
+      { "name" : "sslDebug" },
+      { "name" : "url" },
+      { "name" : "userAgent" },
+      { "name" : "verbose" }
+    ]
+  },
+  {
+    "name" : "okhttp3.curl.Main$VersionProvider",
+    "allDeclaredConstructors" : true,
+    "allPublicConstructors" : true,
+    "allDeclaredMethods" : true,
+    "allPublicMethods" : true
+  },
+  {
+    "name" : "picocli.CommandLine$AutoHelpMixin",
```

Comment:
makes me wonder if there’s a codegen-based alternative!

Reply:
It's meant to work, but since it was out of scope (okcurl) of the main goal (okhttp) of the PR, I went with this as a first version.  I'll follow up with a better fix, and fix logging in okcurl, but that's low priority I think.

Reply:
https://github.com/remkop/picocli/tree/master/picocli-codegen